quail/processes/wps_climdexInput_csv.py

```diff
@@ -136,6 +130,34 @@ def __init__(self):
             status_supported=True,
         )
 
+    def write_csv(self, content):
+        file_ = TempFile(mode="w+", suffix=".csv")
+        file_.write(content)
+        file_.seek(0)
+
+        return file_
```

Comment:
From what I can see we are only using this method in `prepare_csv_files`. If that is the case we can just make it an inner function and save on the `self.` declarations.